integrationTests/testBuiltinFunction.go

```diff
@@ -23,7 +26,23 @@ func (bf *TestBuiltinFunction) IsActive() bool {
 	return true
 }
 
-// IsInterfaceNil --
+// IsInterfaceNil -
 func (bf *TestBuiltinFunction) IsInterfaceNil() bool {
 	return bf == nil
 }
+
+// GenerateOneAddressPerShard -
+func GenerateOneAddressPerShard(shardCoordinator sharding.Coordinator) [][]byte {
+	addresses := make([][]byte, shardCoordinator.NumberOfShards())
+	for i := uint32(0); i < shardCoordinator.NumberOfShards(); i++ {
+		for j := byte(1); j < 255; j++ {
```

Comment:
this can be refactored as you can generate a dummy []byte slice of 32 `1` for all addresses and then just modify the last byte according to the `i` variable. You will need just one for loop.

Reply:
fixed